app/src/main/java/org/wikipedia/settings/PreferenceMultiLine.kt

```diff
@@ -0,0 +1,41 @@
+package org.wikipedia.settings
+
+import android.content.ActivityNotFoundException
+import android.content.Context
+import android.util.AttributeSet
+import android.widget.TextView
+import android.widget.Toast
+import androidx.preference.Preference
+import androidx.preference.Preference.OnPreferenceClickListener
+import androidx.preference.PreferenceViewHolder
+import org.wikipedia.R
+
+class PreferenceMultiLine : Preference {
+    constructor(ctx: Context?, attrs: AttributeSet?, defStyle: Int) : super(ctx, attrs, defStyle)
+    constructor(ctx: Context?, attrs: AttributeSet?) : super(ctx, attrs)
+    constructor(ctx: Context?) : super(ctx)
+
+    override fun onBindViewHolder(holder: PreferenceViewHolder) {
+        super.onBindViewHolder(holder)
+        holder.itemView.findViewById<TextView>(android.R.id.title)?.also {
```

Comment:
Since we are not going to use the return value of the view, would it be better if we use `?.let {}` here?